src/main/java/io/github/pavelapis/minesweeper/Cell.java

```diff
@@ -10,25 +10,30 @@
  * value: [0-9] - number of mined neighbors; -1 - mine.
  */
 class Cell extends JButton {
-
+    public static final long serialVersionUID = 1;
```

Comment:
This must be private and unique inside your application.
See also https://www.baeldung.com/java-serial-version-uid

Reply:
Field 'serialVersionUID' is never used

Reply:
Fixed that